### DISCUSSION TITLE
Add classname on container for previous URL

### DISCUSSION THREAD

**Discussion by kieferslaton** [swup/swup#408] [Help] [answered]
Hey all! New to Swup and trying to get my feet wet. Is there any way I can add a class to the **destination** container before the content is replaced on the page, but only when I'm coming from a specific page?

**Comment by daun** [comment, marked as answer]
swup currently only adds a `to-[path]` class to the `html` element, but if you [look at the source](https://github.com/swup/swup/blob/a93cf10d631e676b01b766fc94666d0fb0a31eba/src/modules/loadPage.js#L44), you can see after which hooks the core library adds and removes them and just repeat that for a `from-[path]` class.
